src/rules/import.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   rules: {
     'import/default': 'off',
     'import/no-default-export': ['warn'],
+    'import/no-named-as-default-member': 'off',
```

Comment:
In welke usecase ben je deze tegengekomen?

Reply:
Misschien eentje om te bespreken dan;
[no-named-as-default-member](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-named-as-default-member.md)

Komt erop neer dat ik in Bothrs codebases vaak zie dat een library in zijn geheel geïmporteerd wordt, om dan members aan te spreken.

We kunnen hier gerust een keuze in maken moving forward ;-) 

Vb
```
import foo from './foo.js';
const bar = foo.bar;

vs 

import { bar } from './foo.js';
```